model/src/main/scala/org/broadinstitute/dsde/rawls/model/StatusModel.scala

```diff
@@ -50,6 +51,7 @@ object Subsystems {
   case object GoogleGenomics extends Subsystem
   case object GoogleGroups extends Subsystem
   case object GooglePubSub extends Subsystem
+  case object Mongo extends Subsystem
```

Comment:
Will this require a version bump for the rawls model lib that orch uses?

Reply:
orch does not *require* a version bump for this. Orch is using the workbench-libs model, not the rawls-model, for `Subsystems`.